src/range_repair.py

```diff
@@ -585,6 +586,10 @@ def _repair_range(options, start, end, step, nodeposition, keyspace=None, column
         repair_status.repair_start(cmd_str, step, start, end, nodeposition, keyspace, column_families)
 
     if not options.dry_run:
+        seconds_to_sleep = randint(0, 60 * 1000) / 1000.0
```

Comment:
Why `* 1000 / 1000.0`? Wouldn't `randint(0, 60)` work?

Reply:
I think Jose's intent here was to decrease the likelihood of collision even further by increasing the potential pool of random start times for the repair while still maintaining a reasonable 60 second window - sample output:
```
31.193
26.756
14.137
38.171
3.922
53.185
```
https://docs.python.org/2.7/library/time.html#time.sleep
>  time.sleep(secs)
>    Suspend execution of the current thread for the given number of seconds. The argument may be a floating point number to indicate a more precise sleep time. 
This function accepts floating point values for increased precision.

Reply:
I doubt we'd really have many collisions, but you can make that simpler with `random.uniform(0, 60)`.

Reply:
Oh neat, that is much simpler. Thanks - updated: https://github.com/onzra/cassandra_range_repair/pull/6/commits/6e98cec8f77dfd8f41267548cdce3dddd74c3e77